db/seeds.rb

```diff
@@ -0,0 +1,23 @@
+if ENV['INTEGRATION_TESTS']
+  puts "Adding user username31"
+  if (User.find_by_username("username31") != nil)
+    puts "User already exists. Please drop the database first"
+    exit -1
+  end
+  u = User.new
+  u.username = "username31"
+  u.email = "a1@b.com"
+  u.password = "test-password"
+  u.admin = true
+  u.save
+
+  puts "Adding namespace for user username31"
```

Comment:
This is not needed, the user namespace is automatically created as soon as the Registry is set.